### PR TITLE
ERMS-2773

### PR DESCRIPTION
**Pull request by Moetez16**
Für EZproxy kann keine IP-Adresse angegeben werden

### CODE REVIEW

File: app/grails-app/controllers/com/k_int/kbplus/AccessPointController.groovy [hunk 1]
```diff
@@ -519,7 +519,7 @@ class AccessPointController extends AbstractDebugController {
                     linkedPlatforms                   : linkedPlatforms,
                     linkedPlatformSubscriptionPackages: linkedPlatformSubscriptionPackages,
                     ip                                : params.ip,
-                    editable                          : true,
+                    editable                          : ((accessService.checkPermAffiliation('ORG_BASIC_MEMBER', 'INST_EDITOR') && orgId == contextService.org.id) || (accessService.checkPermAffiliation('ORG_CONSORTIUM', 'INST_EDITOR'))),
```
Comment (croqueGrec09) [inline app/grails-app/controllers/com/k_int/kbplus/AccessPointController.groovy:522]: set inContextOrg as boolean before the return and then use the variable

File: app/grails-app/controllers/com/k_int/kbplus/AccessPointController.groovy [hunk 2]
```diff
@@ -626,13 +621,14 @@ class AccessPointController extends AbstractDebugController {
...
+                    accessPointDataList               : accessPointDataList,
+                    rgId                              : orgId,
                     platformList                      : orgAccessPoint.getNotLinkedPlatforms(),
                     linkedPlatforms                   : linkedPlatforms,
                     linkedPlatformSubscriptionPackages: linkedPlatformSubscriptionPackages,
-                    ip                                : params.ip, editable: true,
-                    ipv4Ranges                        : ipv4Ranges, ipv4Format: ipv4Format,
-                    ipv6Ranges                        : ipv6Ranges, ipv6Format: ipv6Format,
+                    ip                                : params.ip,
+                    editable                          : ((accessService.checkPermAffiliation('ORG_BASIC_MEMBER', 'INST_EDITOR') && orgId == contextService.org.id) || (accessService.checkPermAffiliation('ORG_CONSORTIUM', 'INST_EDITOR'))),
```
Comment (croqueGrec09) [inline app/grails-app/controllers/com/k_int/kbplus/AccessPointController.groovy:631]: see above

File: app/grails-app/controllers/com/k_int/kbplus/OrganisationController.groovy [hunk 3]
```diff
@@ -1435,23 +1435,26 @@ class OrganisationController extends AbstractDebugController {
...
                             }
                             break
                         case 'ORG_CONSORTIUM':
                             switch (orgInstance.getCustomerType()){
-                                case 'ORG_INST':            isEditable = true; break
+                                case 'ORG_BASIC_MEMBER':    isEditable = userHasEditableRights; break
+                                case 'ORG_INST':            isEditable = userHasEditableRights; break
                                 case 'ORG_CONSORTIUM':      isEditable = false; break
-                                default:                    isEditable = userHasEditableRights; break
+                                default:                    isEditable = false; break
```
Comment (croqueGrec09) [inline app/grails-app/controllers/com/k_int/kbplus/OrganisationController.groovy:1457]: Means a consortium has no editing rights for access point configuration at providers/agencies?

Comment (Moetez16) [reply app/grails-app/controllers/com/k_int/kbplus/OrganisationController.groovy:1457]: Yes.

**Pull request merged**